src/views/Player.vue

```diff
@@ -153,11 +164,44 @@ export default {
       return dateTime
     },
 
+    waitFor(conditionFunction) {
+      const poll = resolve => {
+        if(conditionFunction()) resolve();
+        else setTimeout(() => poll(resolve), 400);
+      }
+      return new Promise(poll);
+    },
+
     logData() {
       if (this.plioId != undefined && this.player.playing) this.uploadJson()
       TIMEOUT = setTimeout(this.logData, UPLOAD_INTERVAL)
     },
 
+    startTheVideo() {
+      var x = document.getElementById('start-button')
+      x.classList.remove('start-button')
+      x.classList.add('start-button-active')
+      setTimeout(() => {
+        this.player.fullscreen.enter();
+        this.waitFor(() => this.player.fullscreen.active === true)
+        .then(() => this.player.play())
+      }, 400);
+
+      this.tutorialProgress['start'] = true;
+      this.checkIfTutorialIsComplete();
+    },
```

Comment:
When start is clicked,
- change the class of the button (so that css changes and we can see the button being "clicked")
- enter fullscreen
- wait till plyr recognizes that fullscreen is active (this was important as otherwise it was giving an exception)
- auto play
- mark the start button pointer of the tutorial as true
- check if the tutorial is complete